recipe/meta.yaml

```diff
@@ -13,9 +13,7 @@ build:
   number: 0
   features:
     - vc9  # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py35]
-    - vc14  # [win and py36]
+    - vc14  # [win and py>=35]
```

Comment:
Could we actually keep the `py35` and `py36` separated as above please? This will make it easier to upgrade to `py37` and drop `py35` down the road. Though, of course, `py34` can be dropped.